detail/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="icon" type="image/x-icon" href="/assets/alchemy-favicon.png" />
+    <script defer src="https://cdn.jsdelivr.net/npm/@supabase/supabase-js"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;600&family=Titillium+Web:wght@200;400;600&display=swap">
+    <link rel="stylesheet" href="/styles/reset.css" />
+    <link rel="stylesheet" href="/styles/global.css" />
+    <link rel="stylesheet" href="/styles/bulletin-board.css" />
+    <title>Community Bulletin Board</title>
+</head>
+
+<body>
+    <header>
+        <img class="logo" src="/assets/cbb-logo.png">
+        <h1>Community BB</h1>
+        <nav>
+            <a href="create-post">Add Post</a>
+            <a id="sign-out-link" href="auth">Sign Out</a>
+        </nav>
+    </header>
+
+    <main>
+        <div id="post-it-detail">
+            <button id="delete-post">Delete post</button>
+            <p id="category"></p>
+            <h2 id="detail-title">Post title</h2>
+            <p id="detail-descript">Post description</p>
+            <p id="detail-contact">contact info</p>
+            <input type="text" id="comment">
+            <button id="add-comment">Post comment</button>
+            <p id="detail-error"></p>
```

Comment:
You can remove this static HTML to clean up the appearance since it will be dynamically rendered and displayed now.